concepts/customization/custom-logo.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Custom Logo
+---
+
+You can customize the Chainlit application with your own logo.
+
+## Use your Logo
+
+Chainlit Application offers support for both dark and light modes. To accommodate this, prepare two versions of your logo, named logo_dark and logo_light. Place these logos in the frontend/public folder. Once you rebuild the frontend and restart the application, your custom logos should be displayed accordingly.
```

Comment:
I think it is just the /public (not frontend/public). This is in the user's app directory, not chainlit source